components/cli/pkg/commands/push.go

```diff
@@ -85,9 +99,11 @@ func RunPush(cellImage string, username string, password string) {
 		if err != nil {
 			util.ExitWithErrorMessage("Failed to push image", err)
 		}
+		pushDockerImages(cellImageMetadata.DockerImages, registryCredentials.Username, registryCredentials.Password)
 	} else {
 		// Pushing image without credentials
 		err = pushImage(parsedCellImage, "", "")
+		pushDockerImages(cellImageMetadata.DockerImages, "", "")
```

Comment:
Shall we move this to an else block under the below if condition ?
Because we only need to push if there are no errors.